src/dotnet/Fable.Compiler/Replacements.fs

```diff
@@ -1342,6 +1342,8 @@ module AstPass =
         | "physicalHash", _ ->
             CoreLibCall("Util", Some "getHashCode", false, i.args)
             |> makeCall i.range i.returnType |> Some
+        | "genericZero", _ -> makeEmit i.range i.returnType [] "0" |> Some
+        | "genericOne", _ -> makeEmit i.range i.returnType [] "1" |> Some
```

Comment:
I try to avoid `emit` in the Replacements module when possible as it prevents optimizations. We also need to check the type, because some classes can have a custom `Zero` static member. I can deal with that, could you please add a comment like `// TODO: Emit Fable AST and check for types with custom zero/one (strings?)`.

Reply:
Got it!

Though I replaced GenericZero/GenericOne and FSharp built-in generic ops with 
```f#
    [<Emit("$0 + $1")>]
    let inline nativeAddition (_x: 'T) (_y: 'U): 'V = jsNative

    [<Emit("$0 / $1")>]
    let inline nativeDivision (_x: 'T) (_y: 'U): 'V = jsNative

    [<Emit("0")>]
    let inline nativeZero<'T> (): 'T = jsNative
```
inside of Array2.fs. ~~Please don't hit me but~~That's a quick workaround to provide non-inlined version of sum's and average's. Not quite sure though to combine both non-inlined and generic zero/ops in Fable. But maybe at least this generic stuff should be moved somewhere else (e.g. to `LanguagePrimitives.Fable` module).

Reply:
Cool! If necessary Fable can inject zero and one as arguments the same way it does with the array constructor.

Reply:
Yes, indeed! Injecting as arguments! `ArrayCons` being injected already should've given me a hint... If it's more feasible than `LanguagePrimitives` with a special treatment in replacements, then I'll go for it in `Lists` module and also make changes in `Array2.fs` also.